monkey/common/configuration/default_agent_configuration.py

```diff
@@ -64,37 +64,38 @@
     8983,
     9200,
     9600,
-]
+)
 
 TCP_SCAN_CONFIGURATION = TCPScanConfiguration(timeout=3.0, ports=TCP_PORTS)
 ICMP_CONFIGURATION = ICMPScanConfiguration(timeout=1.0)
-HTTP_PORTS = [80, 443, 7001, 8008, 8080, 8983, 9200, 9600]
-FINGERPRINTERS = [
+HTTP_PORTS = (80, 443, 7001, 8008, 8080, 8983, 9200, 9600)
+FINGERPRINTERS = (
     PluginConfiguration("elastic", {}),
-    PluginConfiguration("http", {"http_ports": HTTP_PORTS}),
+    # Plugin configuration option contents are not converted to tuples
+    PluginConfiguration("http", {"http_ports": list(HTTP_PORTS)}),
```

Comment:
We can't know the contents of plugin options so we can't freeze it easily. It's also a dict and there's no frozen dict in python.

Reply:
We can wrap `dicts`/`mappings` in `MappingProxyType`. https://docs.python.org/3/library/types.html#types.MappingProxyType

Reply:
^ We can probably skip it for this for now.